lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -199,7 +199,7 @@ def parse_sch(sch_filename, output):
     All clinical records parsed are output to stdout as newline-delimited JSON
     records.  You will likely want to redirect stdout to a file.
     """
-    dtypes = {'census_tract': 'string'}
+    dtypes = {'census_tract': 'str'}
```

Comment:
When left as `string`, reading a `.xls` file with `pd.read_excel` throws the following error:
```
[2020-02-26 13:58:54] Uncaught exception:
Traceback (most recent call last):
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/bin/id3c", line 11, in <module>
    load_entry_point('id3c', 'console_scripts', 'id3c')()
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/kairsten/seattleflu/id3c-customizations/lib/seattleflu/id3c/cli/command/clinical.py", line 206, in parse_sch
    clinical_records = read(sch_filename, dtype=dtypes)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/excel/_base.py", line 334, in read_excel
    **kwds,
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/excel/_base.py", line 885, in parse
    **kwds,
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/excel/_base.py", line 512, in parse
    output[asheetname] = parser.read(nrows=nrows)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/parsers.py", line 1133, in read
    ret = self._engine.read(nrows)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/parsers.py", line 2464, in read
    data = self._convert_data(data)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/parsers.py", line 2534, in _convert_data
    clean_dtypes,
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/parsers.py", line 1727, in _convert_to_ndarrays
    cvals = self._cast_types(cvals, cast_type, c)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/io/parsers.py", line 1825, in _cast_types
    return array_type._from_sequence_of_strings(values, dtype=cast_type)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/core/arrays/string_.py", line 201, in _from_sequence_of_strings
    return cls._from_sequence(strings, dtype=dtype, copy=copy)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/core/arrays/string_.py", line 197, in _from_sequence
    return cls(result)
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/core/arrays/string_.py", line 166, in __init__
    self._validate()
  File "/home/kairsten/.local/share/virtualenvs/id3c-customizations-OmmbKeV1/lib/python3.6/site-packages/pandas/core/arrays/string_.py", line 171, in _validate
    raise ValueError("StringArray requires a sequence of strings or pandas.NA")
ValueError: StringArray requires a sequence of strings or pandas.NA
```